lib/MetaCPAN/Script/Tickets.pm

```diff
@@ -203,6 +205,12 @@ sub parse_tsv {
     return \%summary;
 }
 
+sub rt_dist_url {
+    my ( $self, $dist ) = @_;
+    return 'https://rt.cpan.org/Public/Dist/Display.html?Name='
+        . uri_escape($dist);
+}
+
```

Comment:
We could also have dome something like:

```
    use URI;
    use URI::QueryParam;

    my $uri = URI->new( 'https://rt.cpan.org/Public/Dist/Display.html' );
    $uri->query_param( Name => $dist );
    return $uri;
```

That would also handle the escaping, but TIMTOWDI and it's only one URL param.  Thought I'd point it out, though, since I've found `URI` to be quite handy.